02 ReactApp/src/pods/organization/component.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export class OrganizationComponent extends Component {
+
+  componentDidMount() {
```

Comment:
Going here for class notations because of componentDidMount (we could enhance this but I think is a good idea to keep focus on the talk objectives), once you go for recompose you can wipe this componentDidMount from this component and change it to stateless component). What do you think?

If we go for the table approach for the layout, I would create two child stateless components member >> table, member >> row

What do you think?